apps/heft/src/plugins/CopyFilesPlugin.ts

```diff
@@ -92,6 +95,7 @@ async function _getCopyDescriptorsAsync(
     async (copyConfiguration: ICopyOperation) => {
       // "sourcePath" is required to be a folder. To copy a single file, put the parent folder in "sourcePath"
       // and the filename in "includeGlobs"
+      copyConfiguration.sourcePath = path.resolve(rootFolderPath, copyConfiguration.sourcePath);
```

Comment:
How was this working before? Some implicit resolution of paths, or because the sourcePath got resolved somewhere else?

Reply:
Yeah, I think it must have been implicitly resolving CWD to the `process.cwd` which is always the `buildFolderPath`